spec.html

```diff
@@ -29683,7 +29683,7 @@ <h1>Object.values ( _O_ )</h1>
         <p>This function performs the following steps when called:</p>
         <emu-alg>
           1. Let _obj_ be ? ToObject(_O_).
-          1. Let _nameList_ be ? EnumerableOwnPropertyNames(_obj_, ~value~).
+          1. Let _nameList_ be ? EnumerableOwnProperties(_obj_, ~value~).
```

Comment:
Rename _nameList_ to _valueList_?

Reply:
lol but it's only a value 1.5/3 of the paths :-p

maybe `resultList`?